src/apps/CurrencyConversionApp.jsx

```diff
@@ -41,6 +41,11 @@ const currencies = [
 
 //------------Styled Components-----------------
 
+const Result = styled.div`
+  padding: 0 0 20px 0;
```

Comment:
Let's also try to use units like `rem` for specifying padding or margin so that it's consistent across devices of different sizes.

Reply:
Awesome.